Source/Scene/Scene.js

```diff
@@ -1444,92 +1424,31 @@ define([
-            scene._globeDepth.update(context);
-            scene._globeDepth.clear(context, passState, clearColor);
-        }
-
-        // Determine if there are any translucent surfaces in any of the frustums.
-        var renderTranslucentCommands = false;
-        var frustumCommandsList = scene._frustumCommandsList;
-        var numFrustums = frustumCommandsList.length;
-        for (i = 0; i < numFrustums; ++i) {
-            if (frustumCommandsList[i].indices[Pass.TRANSLUCENT] > 0) {
-                renderTranslucentCommands = true;
-                break;
-            }
-        }
-
-        var clearGlobeDepth = defined(scene.globe) && (!scene.globe.depthTestAgainstTerrain || scene.mode === SceneMode.SCENE2D);
-        var useDepthPlane = clearGlobeDepth && scene.mode === SceneMode.SCENE3D;
-        if (useDepthPlane) {
-            // Update the depth plane that is rendered in 3D when the primitives are
-            // not depth tested against terrain so primitives on the backface
-            // of the globe are not picked.
-            scene._depthPlane.update(frameState);
-        }
-
-        // If supported, configure OIT to use the globe depth framebuffer and clear the OIT framebuffer.
-        var useOIT = !picking && renderTranslucentCommands && defined(scene._oit) && scene._oit.isSupported();
-        if (useOIT) {
-            scene._oit.update(context, scene._globeDepth.framebuffer);
-            scene._oit.clear(context, passState, clearColor);
-            useOIT = useOIT && scene._oit.isSupported();
-        }
-
-        // If supported, configure FXAA to use the globe depth color texture and clear the FXAA framebuffer.
-        var useFXAA = !picking && scene.fxaa;
-        if (useFXAA) {
-            scene._fxaa.update(context);
-            scene._fxaa.clear(context, passState, clearColor);
-        }
-
-        if (sunVisible && scene.sunBloom) {
-            passState.framebuffer = scene._sunPostProcess.update(context);
-        } else if (useGlobeDepthFramebuffer) {
-            passState.framebuffer = scene._globeDepth.framebuffer;
-        } else if (useFXAA) {
-            passState.framebuffer = scene._fxaa.getColorFramebuffer();
-        }
-
-        if (defined(passState.framebuffer)) {
-            clear.execute(context, passState);
-        }
-
         // Ideally, we would render the sky box and atmosphere last for
         // early-z, but we would have to draw it in each frustum
         frustum.near = camera.frustum.near;
         frustum.far = camera.frustum.far;
         us.updateFrustum(frustum);
 
+        var skyBoxCommand = scene._state.skyBoxCommand;
         if (defined(skyBoxCommand)) {
             executeCommand(skyBoxCommand, scene, context, passState);
         }
 
-        if (defined(skyAtmosphereCommand) && skyAtmosphereVisible) {
-            executeCommand(skyAtmosphereCommand, scene, context, passState);
+        if (scene._state.isSkyAtmosphereVisible) {
```

Comment:
This function could be cleaned up by assign `scene._state` to a local.  Same for a few other places in this file.